From: David Milburn <dmilburn@redhat.com>
Date: Thu, 18 Nov 2010 20:40:53 -0500
Subject: [misc] pci: update Intel Patsburg defines
Message-id: <1290112856-1796-7-git-send-email-dmilburn@redhat.com>
Patchwork-id: 29506
O-Subject: [RHEL5.6 PATCH 6/9] BZ 570868 PCI: update Intel Patsburg defines
Bugzilla: 570868
RH-Acked-by: Stanislaw Gruszka <sgruszka@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Michal Schmidt <mschmidt@redhat.com>

commit cb04e95bdd0bfd618ab731c84a3ab56b56974df8
Author: Seth Heasley <seth.heasley@intel.com>
Date:   Mon Oct 4 13:27:14 2010 -0700

    PCI: update Intel chipset names and defines

BZ 570868

diff --git a/arch/i386/pci/irq.c b/arch/i386/pci/irq.c
index b0cd69d..74c0c6d 100644
--- a/arch/i386/pci/irq.c
+++ b/arch/i386/pci/irq.c
@@ -555,7 +555,7 @@ static __init int intel_router_probe(struct irq_router *r, struct pci_dev *route
 		case PCI_DEVICE_ID_INTEL_ICH10_1:
 		case PCI_DEVICE_ID_INTEL_ICH10_2:
 		case PCI_DEVICE_ID_INTEL_ICH10_3:
-		case PCI_DEVICE_ID_INTEL_PBG_LPC:
+		case PCI_DEVICE_ID_INTEL_PATSBURG_LPC:
 			r->name = "PIIX/ICH";
 			r->get = pirq_piix_get;
 			r->set = pirq_piix_set;
diff --git a/include/linux/pci_ids.h b/include/linux/pci_ids.h
index 4ffe59e..79b4b47 100644
--- a/include/linux/pci_ids.h
+++ b/include/linux/pci_ids.h
@@ -2220,7 +2220,7 @@
 #define PCI_DEVICE_ID_INTEL_CPT_SMBUS	0x1c22
 #define PCI_DEVICE_ID_INTEL_CPT_LPC_MIN	0x1c41
 #define PCI_DEVICE_ID_INTEL_CPT_LPC_MAX	0x1c5f
-#define PCI_DEVICE_ID_INTEL_PBG_LPC	0x1d40
+#define PCI_DEVICE_ID_INTEL_PATSBURG_LPC	0x1d40
 #define PCI_DEVICE_ID_INTEL_82801AA_0	0x2410
 #define PCI_DEVICE_ID_INTEL_82801AA_1	0x2411
 #define PCI_DEVICE_ID_INTEL_82801AA_3	0x2413
